docs/src/content/docs/fr/index.mdx

```diff
@@ -39,19 +41,122 @@ import AboutAstro from '~/components/about-astro.astro';
+
+		Sa conception intuitive permet non seulement de rationaliser mon flux de travail, mais aussi de réduire le temps d'intégration des développeurs de logiciels libres.
+
+	</Testimonial>
+	<Testimonial
+		name="Joel"
+		handle="jhooks"
+		cite="https://twitter.com/jhooks/status/1727405160547418405"
+	>
+		J'ai passé un peu plus de temps avec Starlight pour la documentation du Course Builder et c'est très bien jusqu'à présent. Il y a beaucoup de touches agréables et je peux me concentrer sur l'écriture en Markdown et non sur la manipulation du site.
+	</Testimonial>
+	<Testimonial
+		name="Rick"
+		handle="rick_viscomi"
+		cite="https://twitter.com/rick_viscomi/status/1665867447910510593"
+	>
+		J'ai commencé à jouer avec Starlight. Je dois dire que je suis très impressionné par ses performances.
+
+		💯💯💯💯
+
+	</Testimonial>
+	<Testimonial
+		name="Nicolas"
+		handle="beaussan"
+		cite="https://twitter.com/beaussan/status/1735625189583466893"
+	>
+		Starlight est la meilleure façon de commencer avec la documentation, entre la puissance et la vitesse d'Astro et l'outillage de Starlight, c'est la meilleure solution.
+		la puissance et la vitesse d'Astro, et l'outillage de Starlight, c'est une combinaison paradisiaque.
+
+		C'est mon outil de prédilection depuis un certain temps, et je continue à l'aimer !
+
+	</Testimonial>
+	<Testimonial
+		name="Sylwia"
+		handle="SylwiaVargas"
+		cite="https://x.com/SylwiaVargas/status/1726556825741578286"
+	>
+		J'ai utilisé Starlight dans mon dernier emploi et je l'ai adoré. Excellents composants, conception intuitive
+		intuitif, et une communauté super réactive (chaque fois que quelqu'un avait besoin de quelque chose, ils le livraient rapidement ou vous proposaient une solution de contournement),
+		ils vous le livraient rapidement ou vous proposaient une solution de contournement). Une expérience très agréable.
+	</Testimonial>
+	<Testimonial
+		name="Lou Cyx"
+		handle="loucyx"
+		cite="https://elk.zone/m.webtoo.ls/@loucyx@mastodon.social/111587380021362284"
+	>
+		La documentation de mon site monorepo sont plus beaux que jamais grâce à Starlight. C'est extrêmement facile à utiliser sans perdre toute la puissance d'Astro. Merci d'y avoir travaillé !
+	</Testimonial>
+	<Testimonial
+		name="BowTiedWebReaper"
+		handle="BowTiedWebReapr"
+		cite="https://twitter.com/BowTiedWebReapr/status/1735633399501697517"
+	>
+		Starlight est mon outil de référence pour la documentation. Il m'a permis d'ajouter facilement des documents à mon site web Astro existant, sans avoir besoin d'un sous-domaine à utiliser avec un autre outil.
+	</Testimonial>
+	<Testimonial
+		name="Jeff"
+		handle="J_Everhart383"
+		cite="https://twitter.com/J_Everhart383/status/1691900590048292908"
+	>
+		J'ai reconstruit la documentation de la plateforme WPEngine Atlas. Croyez-moi quand je dis que Starlight a tout ce qu'il faut pour faire une plateforme A+ de documentation 🙌
```

Comment:
```suggestion
		J'ai reconstruit la documentation de la plateforme WPEngine Atlas. Croyez-moi quand je dis que Starlight a tout ce qu'il faut pour faire une plateforme de documentation 3 étoiles 🙌
```

The A+ refers to the grade system which is not well known by many people speaking French I think and using 20/20 would sound weird I think. Any thoughts about using "3 étoiles" instead?

Reply:
It's true that this is less common in France, so let's use stars as a rating in this case!